vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -395,6 +350,11 @@ def update_room(self, vc_room, event):
         if vc_room.data['mute_host_video'] == zoom_meeting_settings['host_video']:
             changes.setdefault('settings', {})['host_video'] = not vc_room.data['mute_host_video']
 
+        alternative_hosts = process_alternative_hosts(zoom_meeting_settings.get('alternative_hosts', []))
```

Comment:
Isn't the default value of `zoom_meeting_settings.get('alternative_hosts')` supposed to be a string to be parsed in `process_alternative_hosts`?

Reply:
I think `zoom_meeting_settings` is what we store in the indico db (VCRoom.data), and the comma-separated string is what's used on the Zoom API side.

Reply:
Indeed. I confused it with the data which gets stored in the room's data. Thanks!